app-full-microprofile/threshold.properties

```diff
@@ -1,6 +1,6 @@
-linux.jvm.time.to.first.ok.request.threshold.ms=2400
+linux.jvm.time.to.first.ok.request.threshold.ms=2500
```

Comment:
This change is not mentioned in the description and it is not related to name refactoring. Any specific reason for changing this threshold?